logging/zap/grpclogger.go

```diff
@@ -44,3 +45,80 @@ func (l *zapGrpcLogger) Printf(format string, args ...interface{}) {
 func (l *zapGrpcLogger) Println(args ...interface{}) {
 	l.logger.Info(fmt.Sprint(args...))
 }
+
+// GrpcLoggerV2Options allows you to set grpclogger options when using ReplaceGrpcLoggerV2.
+type GrpcLoggerV2Options struct {
+	verbosity int
```

Comment:
This is not accessible :( you would never be able to set this.

Reply:
Oh whoops lol :(